DigitalLearningSolutions.Web/Styles/myAccount/myAccount.scss

```diff
@@ -0,0 +1,24 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+$nhs-dark-grey: #425563;
+
+.account-card-content {
+  padding-top: 16px;

```

Comment:
Super minor things and more of a FYI:
* In CSS you can specify padding like this: padding: 16px 32px 16px 32px; (this is in order of top right bottom left). If you leave out a particular value, it default to the opposite side, e.g. padding: 16px 32px; would be equivalent to the previous one. The only exception is padding: 16px; which would set 16px padding on all sides.
I personally prefer that syntax as it's more concise and keeps the padding properties in one line.
* 0 doesn't need units so I'd probably keep it consistent across the file.
* I'd have a preference for "border: none" as it's more intuitive, but I think 0 is fine too! :)